.changelog/21524.txt

```diff
@@ -0,0 +1,3 @@
+```release-note:security
+Upgrade envoy module dependencies to version 1.27.7, 1.28.5 and 1.29.7 or higher to resolve CVE-2024-39305
```

Comment:
Thanks a lot for the PR, this new way of bumping Envoy is awesome! 🚀 
LGTM, just one small suggestion on the change log.

```suggestion
Upgrade envoy module dependencies to version 1.27.7, 1.28.5 and 1.29.7 or higher to resolve [CVE-2024-39305](https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2024-39305)
```